File: app/src/main/java/com/devserrano/dragonballexplorer/database/FavoriteCharacter.java
package com.devserrano.dragonballexplorer.database;

import androidx.annotation.NonNull;
import androidx.room.Entity;
import androidx.room.PrimaryKey;

@Entity(tableName = "favorite_characters")
public class FavoriteCharacter {

    @PrimaryKey
    public int id;

    @NonNull
    public String name;

    @NonNull
    public String race;

    @NonNull
    public String ki;

    @NonNull
    public String image;

    @NonNull
    public String note;

    public FavoriteCharacter(
            int id,
            @NonNull String name,
            @NonNull String race,
            @NonNull String ki,
            @NonNull String image,
            @NonNull String note
    ) {
        this.id = id;
        this.name = name;
        this.race = race;
        this.ki = ki;
        this.image = image;
        this.note = note;
    }
}

File: app/src/main/java/com/devserrano/dragonballexplorer/database/FavoriteCharacter.kt
package com.devserrano.dragonballexplorer.database
import androidx.room.Entity
import androidx.room.PrimaryKey


@Entity(tableName = "favorite_characters")
data class FavoriteCharacter (

    @PrimaryKey
    val id: Int,
    val name: String,
    val race: String,
    val ki: String,
    val image: String,
    val note: String = ""


)


